.github/workflows/r_valgrind.yml

```
@@ -32,13 +32,14 @@ jobs:
           ref: "refs/pull/${{ github.event.client_payload.pr_number }}/merge"
       - name: Send init status
         if: ${{ always() }}
+        shell: bash
         run: |
           $GITHUB_WORKSPACE/.ci/set-commit-status.sh \
             "${{ github.workflow }}" \
             "pending" \
             "${{ github.event.client_payload.pr_sha }}"
           comment="Workflow **${{ github.workflow }}** has been triggered! 🚀\r\n"
-          comment+="${GITHUB_SERVER_URL}/microsoft/LightGBM/actions/runs/${GITHUB_RUN_ID}"
+          comment="${comment} ${GITHUB_SERVER_URL}/microsoft/LightGBM/actions/runs/${GITHUB_RUN_ID}"
```

I don't think that we should start a new line with a whitespace character (note that previous line ends with `\r\n`).
```suggestion
          comment="${comment}${GITHUB_SERVER_URL}/microsoft/LightGBM/actions/runs/${GITHUB_RUN_ID}"
```

---

ah right! agree